cocos/2d/CCNode.h

```diff
@@ -142,6 +143,15 @@ class CC_DLL Node : public Ref
     /// @{
     /// @name Setters & Getters for Graphic Properties
 
+    /*
+    setZLocalOrderOfArrive totally control the render order, it use for sometime node should cover the friendnode and sometime under the friendnode
+    */
+    void setZLocalOrderOfArrive(std::int64_t nZOrderArrived);
```

Comment:
i should change the render seq. like tab view button. clk is cover the bg, no clk is under the bg.
if it can control the render seq it is the easy way to dev